src/Console/Arguments.php

```diff
@@ -164,6 +164,7 @@ public function getOption(string $name): string|bool|null
     /**
      * Get a boolean option's value or null if not set.
      *
+     * @param string $name Name
```

Comment:
I would go the opposite way and remove such redundant annotations everywhere :)

Reply:
Right now it is consistent to have them all matching.
It also increases readability (as the order relates to where to find them).
If we would drop them randomly, things go boom soon.
As you can see in other places, they might also get mixed up in order when adding.

So overall I am not in favor of removing them.